README.md

```diff
@@ -1,289 +1,137 @@
 # first-microverse-project
 <a name="readme-top"></a>
 
-<!--
-HOW TO USE:
-This is an example of how you may give instructions on setting up your project locally.
-
-Modify this file to match your project and remove sections that don't apply.
-
-REQUIRED SECTIONS:
-- Table of Contents
-- About the Project
-  - Built With
-  - Live Demo
-- Getting Started
-- Authors
-- Future Features
-- Contributing
-- Show your support
-- Acknowledgements
-- License
-
-OPTIONAL SECTIONS:
-- FAQ
-
-After you're finished please remove all the comments and instructions!
--->
-
-<div align="center">
-  <!-- You are encouraged to replace this logo with your own! Otherwise you can also remove it. -->
-  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
-  <br/>
-
-  <h3><b>Microverse README Template</b></h3>
-
-</div>
-
 <!-- TABLE OF CONTENTS -->
 
 # 📗 Table of Contents
 
 - [📖 About the Project](#about-project)
   - [🛠 Built With](#built-with)
-    - [HTML & CSS](#tech-stack)
+    - [Tech stack](#tech-stack)
     - [Key Features](#key-features)
-  - [🚀 Live Demo](#live-demo)
 - [💻 Getting Started](#getting-started)
-  - [Setup](#setup)
```

Comment:
- [ ] [OPTIONAL] I have noticed that in the current commits, you have deleted the setup navigation. However, you should remember that every project repository requires setup and installation procedures ☺️ . Therefore, I suggest considering rewriting the navigation for the professional readme file.
- [ ] [OPTIONAL] Again, the above comment is supported by this. It would be of great importance to ensure that your readme has the setup and installation procedures. This enables other developers, to have a clear understanding of the processes he has to go through to have the project set on a local machine 👍 .